webui/react/src/pages/SignIn.tsx

```diff
@@ -18,7 +18,6 @@ import LogoOkta from 'shared/assets/images/logo-sso-okta-white.svg';
 import { StoreActionUI } from 'shared/contexts/UIStore';
```

Comment:
These changes seem unrelated?

Reply:
true. it's related to what I was chatting with Keita about but not this. just being lazy about making two PRs for something that __should__ essentially be a no-op. it would be safer to have two separte ones..

Reply:
I went ahead and split that into its own pr